piker/brokers/kraken.py

```diff
@@ -570,8 +638,34 @@ async def stream_quotes(
 
                 elif typ == 'l1':
                     quote = ohlc
-                    topic = quote['symbol']
+                    topic = quote['symbol'].lower()
 
                 # XXX: format required by ``tractor.msg.pub``
                 # requires a ``Dict[topic: str, quote: dict]``
                 await send_chan.send({topic: quote})
+
+
+@tractor.context
+async def open_symbol_search(
+    ctx: tractor.Context,
+) -> Client:
```

Comment:
This return typing is probably wrong, not sure how we want that yet in `tractor`.